packages/stable/src/exports.gen.ts

```diff
@@ -116,6 +116,20 @@ export {
   Label,
   LabelList
 } from './api/documents/types.gen';
+export {
+  CursorQueryParameter,
+  IdentityType,
+  IdentityTypeFilter,
+  UserIdentifier,
+  UserProfileItem,
+  UserProfilesByIdsRequest,
+  UserProfilesByIdsResponse,
+  UserProfilesErrorResponse,
+  UserProfilesListResponse,
+  UserProfilesNotFoundResponse,
```

Comment:
it seems like some of these types aren't needed/used, @andersfylling is there a setting to omit them somehow?

Reply:
Hm, I don't think we added support for that. Question here is why aren't they using an ErrorResponse type in the open api spec?